app/utils/bundle-importer.js

```diff
@@ -252,8 +260,17 @@ YUI.add('bundle-importer', function(Y) {
     */
     _execute_addMachines: function(record, next) {
       var parentKey = record.args[0].parentId;
+      parentKey = parentKey && parentKey.replace(/^\$/, '');
```

Comment:
all 'key' references in the recordSet returned from bundlelib use that as an indicator that the field needs to be replaced by a value found in the specified keys record.